src/components/Forms/Register-Customer.js

```diff
@@ -0,0 +1,206 @@
+import React, { Component } from 'react';
+
+class RegisterCustomer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      fName: '',
+      lName: '',
+      email: '',
+      phone: '',
+      zipcode: '',
+      address: '',
+      city: '',
+      state: '',
+      country: '',
+      photo_url: '',
+      //   password: '',
+    };
+  }
+
+  handleSubmit = event => {
+    event.preventDefault();
+    //   testing the handlesubmit
+    const data = this.state;
+    console.log('Final Data', data);
+  };
+
+  handleInputChange = event => {
+    event.preventDefault();
+    //   getting name of input and value
+    // console.log('handleINputChange Name', event.target.name);
+    //   console.log('handleINputChange Value', event.target.value);
+    this.setState({
+      // seting key to key-value pair
+      [event.target.name]: event.target.value,
+    });
+  };
+
+  // must be async in order to await key function
+  async postDataGroomers() {
+    try {
+      let result = await fetch(
```

Comment:
We do have Axios installed as a dependency and you can use that as an alternative to fetch(). You also wouldn't have to stringify the data, as Axios will do that